autopush-common/Cargo.toml

```
@@ -65,7 +62,7 @@ form_urlencoded = { version = "1.2", optional = true }
 [dev-dependencies]
 mockito = "0.31"
 tempfile = "3.2.0"
-tokio = { version = "0.2", features = ["macros"] }
+tokio = { version = "1.38", features = ["macros"] }
```

```suggestion
tokio = { workspace=true, features = ["macros"] }
```

This will pull the version from the main Cargo.toml file, so we don't run the risk of discrepancies later.

---

Great point!